tests/python/frontend/tflite/test_forward.py

```diff
@@ -901,79 +859,6 @@ def test_all_resize():
     if 'RESIZE_NEAREST_NEIGHBOR' in dir(BuiltinOperator()):
         _test_resize(tf.image.resize_nearest_neighbor, data, align_corners=False)
 
-#######################################################################
-# Range
-# -----
-def _test_range(start, limit, delta):
```

Comment:
_test_range why removed?  i think this is lost during resolving conflict.

Reply:
Fixed. Got removed during resolving conflicts.